src/gsCore/gsMultiPatch.h

```diff
@@ -226,12 +226,18 @@ class gsMultiPatch : public gsBoxTopology, public gsFunctionSet<T>
 
     using Base::addInterface; // unhide base function
     
-    /// Add side s of patch g to the outer boundary of the domain
+    /// Add side \a s of patch \a g to the outer boundary of the domain
     void addPatchBoundary( gsGeometry<T>* g, boxSide s ) {
         int p = findPatchIndex( g );
         gsBoxTopology::addBoundary( patchSide( p, s ) );
     }
 
+    /// Get coordinates of the patchCorner \a pc in the physical domain
+    gsMatrix<T> pointOn( const patchCorner& pc );
+
+    /// Get coordinates of the midpoint of the the patchSide \a ps in the physical domain
```

Comment:
"of the midpoint of the"  --> is misleading, maybe "of a 'central' point on the patchSide"
the the --> the